UDPClientGUI/Events/ClientStatusChangedEvent.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace UDPCommGUI
+{
+    class ClientStatusChangedEvent
+    {
+        public ClientStatusChangedEvent(bool status)
+        {
+            this.Status = status;
+        }
+
+        public bool Status { get; private set; }
```

Comment:
Checkout readonly properties. I know I said to make it `private set` but since the value is only set in the constructor, you can omit the set entirely.

This is only available in the later versions of C# (I believe C# 8.0+)

Reply:
Implemented